src/pages/static_page.ts

```diff
@@ -20,6 +20,7 @@ createVueApp( App,
 		bucketClasses: bucketIdToCssClass( pageData.selectedBuckets ),
 		pageIdentifier: PAGE_IDENTIFIER,
 		page: StaticPage,
+		pageTitle: 'static_page_title',
```

Comment:
So this won't work, because we have multiple static pages and we need a way to dynamically get the key for them.

It's also a little confusing because there's another `pageTitle` parameter that's used for the heaing of the page. We should discuss this in the daily.

Reply:
Okay, I added the page titles to the messages in https://github.com/wmde/fundraising-application/pull/2953

This means you can now pass the pageId as the title and it'll get set correctly

`pageTitle: staticPage.getAttribute( 'data-page-id' )`

Reply:
I did this - `pageTitle: staticPage.getAttribute( 'data-page-id' )` change in the following three files, because they had the same issue.

- src/pages/privacy_protection.ts
- src/pages/static_page.ts
- src/pages/supporters.ts